src/components/layout/TitleContainer.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+const TitleContainer = styled.div`
+  height: 15vh;
+  padding: 2rem;
+  
+  h1 {
+    color: #084B83;
+  }
+  
+  p {
+    color: #FF66B3
+  }
```

Comment:
Think about starting a styling guide with the app colours and variables!

Reply:
An Issue was created and be implemented soon: #18